### PR TITLE
Release/1.11.0

### PR DESCRIPTION
**Pull request by joegasewicz**
###  #996: Implement new areas MVP page design

**Start page:**

- Ensure content is updated as per Figma designs.
- Add related contents section, we don't have URLs yet for the links, so happy for this to be static text for now. 
- Amend Scotland and NI links, these should now point to:
 - Scotland: https://statistics.gov.scot/atlas/resource?uri=http://statistics.gov.scot/id/statistical-geography/S92000003
 - NI: https://www.ninis2.nisra.gov.uk/public/AreaProfile.aspx?Menu=True

**Overview page:**

- Ensure content is updated as per Figma designs.
- Feature flag/remove map and update designs as per Figma.
- Add related contents section, we don't have URLs yet for the links, so happy for this to be static text for now. 
- Add section for Changes over time tool (again, URL is not yet available so static text can be used). 

### Who can review
anyone 

Describe who worked on the changes, so that other people can review.
myself

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ First build the client assets
 Run the main server
 * Run `make debug`
 
+To develop the frontend assets (css, js, ts etc.) then the following 2 make tasks will watch any file changesin the `assets/dist` directory & your frontend will reflect these updates on page reload (you might need to clear your cache withL `⌘ + R`):
```
Comment (nshumoogum) [inline README.md:13]:
```suggestion
To develop the frontend assets (css, js, ts etc.) then the following 2 make targets will watch any file changes in the `assets/dist` directory & your frontend will reflect these updates on page reload (you might need to clear your cache withL `⌘ + R`):
```

File: assets/locales/service.cy.toml [hunk 2]
```diff
@@ -15,95 +15,98 @@
...
-
 [OtherCountries]
 description = "Other countries"
 one = "Other countries"
 
-[FindFactsAndFiguresScotland]
-description = "Find facts and figures about areas in Scotland on Scotland’s Census website"
-one = "Find facts and figures about <a href=\"https://www.scotlandscensus.gov.uk/search-the-census#/explore\">areas in Scotland on Scotland’s Census website</a>."
+[OtherCountriesContent]
+description = "Scotland and Northern Ireland have their own agencies who produce official statistics. View facts and figures for areas in:"
```
Comment (nshumoogum) [inline assets/locales/service.cy.toml:29]: Are we not looking to add welsh content?

File: config/config.go [hunk 3]
```diff
@@ -17,6 +17,7 @@ type Config struct {
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
+	ShowONSMap                 bool
```
Comment (nshumoogum) [inline config/config.go:20]: Probably want to add in the envconfig tag to be able to pick an environment variable to effectively change ShowONSMap from false to true or vice versa.

We usually stick with the naming convention:

```suggestion
	EnableONSMap                 bool       `envconfig:"ENABLE_ONS_MAP"`
```

Default setting should be false

File: webpack/webpack.config.js [hunk 4]
```diff
@@ -14,10 +15,10 @@ module.exports = (args = { mode: "production", analyze: false }) => {
         entry: {
             areaLanding: {
                 import: path.resolve(__dirname, "../public/ts/area-landing.ts"),
-                dependOn: "mapboxGL"
+                // dependOn: "mapboxGL"
```
Comment (nshumoogum) [inline webpack/webpack.config.js:18]: Is it worth removing these comments altogether

**Review by nshumoogum — APPROVED** [review]
I've made suggestions, possibly could follow these up in a separate pull request if you wanted to release this

File: webpack/webpack.config.js (hunk 4 above)
Comment (joegasewicz) [reply webpack/webpack.config.js:18]: Yep, these will all be added back in at a later date hopefully

File: config/config.go (hunk 3 above)
Comment (joegasewicz) [reply config/config.go:20]: Most of the map logic has been removed now, so this variables isn't being used, so i can just remove it.

File: assets/locales/service.cy.toml (hunk 2 above)
Comment (joegasewicz) [reply assets/locales/service.cy.toml:29]: I think that's coming, i was told to not use google translate myself lol

**Pull request merged**